sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3013,6 +3013,16 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       sql("reset")
     }
   }
+
+  test("clear cache command should not fail with cache") {
+    withTable("tbl") {
+      val provider = spark.sessionState.conf.defaultDataSourceName
+      sql(s"CREATE TABLE tbl(i INT, j STRING) USING $provider")
+      sql("clear cache")
+      sql("cache table tbl")
+      sql("clear cache")
+    }
+  }
```

Comment:
This test case doesn't fail without the patch. Please remove this if this is not your intention.

Reply:
ok